crates/loro-wasm/src/lib.rs

```diff
@@ -2120,8 +2175,8 @@ impl LoroTree {
     /// const subscription = tree.subscribe((event)=>{
     ///     console.log(event);
```

Comment:
It'll be better if we can describe the event here since `Tree` is a very special one